config/locales/views/submissions/en.yml

```diff
@@ -78,7 +81,12 @@ en:
       no_submission: You have not submitted any work yet.
       only_required_files: You may only submit the required files listed above.
       select_file: Select a file to view
+      url: URL
+      url_text: Display URL as
       version_control_warning: Submitting files here overrides any changes made using version control. Make sure you have worked on the most recent files before submitting.
     submission_summary: Submission Report
     successfully_changed: Successfully updated %{changed} results.
     unrelease_marks: Unrelease Marks
+    unsupported_url: Previews from %{host} are not supported and cannot be displayed
```

Comment:
This message is a bit misleading, since it's a catch-all message for when any error occurs. For example, if there's a youtube link but it's invalid.

For now, let's just replace it with `url_display_error` below. (At a later point, perhaps we can change this so it's only displayed when there' an invalid `hostname`, but that's not necessary for this PR.)